### DISCUSSION TITLE
Leveraging Free AI with Ollama in .NET Aspire

### DISCUSSION THREAD

**Discussion by giscus** [VladislavAntonyuk/vladislavantonyuk.github.io#115] [Announcements]
# Leveraging Free AI with Ollama in .NET Aspire

https://vladislavantonyuk.github.io/articles/Leveraging-Free-AI-with-Ollama-in-.NET-Aspire/

**Comment by GeorgeLeithead** [comment]
Really great article and I like the integration with Aspire!
There are a lot of 'magic numbers' here.  Such as the Tag is step 3, the version in step 5, etc.  Would be really helpful if you expanded and let us know what these are and how to get them (especially if reading this and a newer version is available).

**Reply by VladislavAntonyuk** [reply]
Thank you @GeorgeLeithead!
As for step 3, I will update the article and provide the link to docker image.
As for step 5, I added info, that 5 minutes is an experimental value, because response from llama3 is really slow and because of default Aspire configuration resilience (30 sec timeout), I always receive exception. I will play with different models, probably they will provide better results.
